adapter/internal/discovery/xds/rate_limiter_cache.go

```diff
@@ -88,6 +88,7 @@ func (r *rateLimitPolicyCache) AddAPILevelRateLimitPolicies(apiID string, mgwSwa
 		return nil
 	}
 	level := strings.ToUpper(mgwSwagger.RateLimitLevel)
+	loggers.LoggerXds.Infof("Rate-limiting enabled for the API: %q, level: %q", apiID, level)
```

Comment:
This is not logged during API calls right? Logged only during a deployment?

Reply:
Yes, during deployment

Reply:
This method is being called during the API deployment time. Hence log will not be printed for the API invocations.